main.py

```diff
@@ -1,10 +1,46 @@
 import init_django_orm  # noqa: F401
-
+import json
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json", "r") as file:
+        players_data = json.load(file)
+
+    for player_data in players_data:
+        user = players_data[player_data]
```

Comment:
Directly accessing dictionary keys without using `.get()` can lead to `KeyError` if the key is not present. Consider using `.get()` to safely access dictionary values, as suggested in the previous review.